### DISCUSSION TITLE
how to recognize Julia about cmdstan executable?

### DISCUSSION THREAD

**Discussion by YutaroIshihama** [StanJulia/Stan.jl#138] [Q&A] [answered]
I am a windows 10 user and believe make and build of the `cmdstan` is successfully done, because so-called the Bernoulli checking is done well. I created the cmdstan directory under d:/cmdstan by git clone. Trial of `binomial.jl` located under d:/cmdstan directory after I set the variable `ENV["CMDSTAN"] = "D:\\cmdstan"` from the Julia 1.11 REPL complains the following message. Please advice how to work it around. Thanks in advance for your kind advice.

`LoadError: IOError: cd(""): invalid argument (EINVAL)`

**Comment by goedman** [comment]
Hi, is d://cmdstan the directory where you installed the cmdstan binaries (from where you ran the Bernoulli check?)

**Reply by YutaroIshihama** [reply]
thx for reply. After posting this, I have switched to `cmdstanpy`. That is why I am replying so lately. If one creates executable file by `make` in the same directory where `stan` file is sitting, the relevant problem would be solved, which I have not tested yet.

**Reply by goedman** [reply, marked as answer]
Great, I never tested StanJulia with cmdstanpy so I don't know if that will work but there maybe other options to run cmdstanpy programs from Julia.
